src/config/workload.rs

```diff
@@ -44,22 +47,33 @@ impl Workload {
 #[derive(Clone, Deserialize)]
 pub struct Topics {
     topics: usize,
+    #[serde(default = "one")]
+    partitions: usize,
     topic_len: usize,
     message_len: usize,
+    #[serde(default = "four")]
+    key_len: usize,
```

Comment:
Should there be any constraint on number of keys like we allow for keyspace? Why is 4 the default?

Reply:
Since we explicitly set the partition when producing one message, we don't need to control the keys, as long as there is a key, it's OK. 4 is just a random number, I can change it to at least one.